test/llvm_autogenerated/invoke_wrapper.s

```diff
@@ -0,0 +1,201 @@
+	.text
```

Comment:
For .s files which are generated from C/C++ files (i.e. not small and hand-written),  let's also check in the C/C++ source file, so the .s file can be easily regenerated if there is a major change. I've been checking them in right alongside the .s files in `test/dot_s`. Also this `llvm_autogenerated` directory is for tests which are generated from the .ll test cases from the wasm backend. We use `test/dot_s` for other tests, including those that are generated manually from C files. So this test (assuming you did manually generate it from a C file or hand write it) should go in `test/dot_s`.